stellar/operators.js

```diff
@@ -1,127 +1,177 @@
+const { Command } = require('commander');
+const { loadConfig, printInfo, printWarn, parseArgs, validateParameters, saveConfig } = require('../evm/utils');
+const { getWallet, broadcast, addBaseOptions, addressToScVal, tokenToScVal } = require('./utils');
+const { addOptionsToCommands, getChainConfig } = require('../common');
+const { prompt } = require('../common/utils');
+
+async function isOperator(wallet, _, chain, contract, args, options) {
+    const [address] = args;
+    const operation = contract.call('is_operator', addressToScVal(address));
+    const result = await broadcast(operation, wallet, chain, 'is_operator called', options);
+
+    if (result.value()) {
+        printInfo(address + ' is an operator');
+    } else {
+        printWarn(address + ' is not an operator');
+    }
+}
 
-    const contract = new Contract(options.address || chain.contracts?.axelar_operators?.address);
+async function addOperator(wallet, _, chain, contract, args, options) {
+    const [address] = args;
+    const operation = contract.call('add_operator', addressToScVal(address));
+    await broadcast(operation, wallet, chain, 'add_operator called', options);
+}
 
-    let operation;
+async function removeOperator(wallet, _, chain, contract, args, options) {
+    const [address] = args;
+    const operation = contract.call('remove_operator', addressToScVal(address));
+    await broadcast(operation, wallet, chain, 'remove_operator called', options);
+}
 
-    switch (options.action) {
-        case 'is_operator': {
-            if (!options.args) {
-                throw new Error(`Missing --args operatorAddress the params.`);
-            }
+async function addGas(wallet, _, chain, contract, args, options) {
+    const operator = addressToScVal(wallet.publicKey());
+    const [sender, messageId, spender, tokenAddress, tokenAmount] = args;
 
-            const operator = Address.fromString(options.args).toScVal();
-            operation = contract.call('is_operator', operator);
-            break;
-        }
+    validateParameters({
+        isNonEmptyString: { sender, messageId, spender, tokenAddress },
+        isValidNumber: { tokenAmount },
+    });
 
-        case 'add_operator': {
-            if (!options.args) {
-                throw new Error(`Missing --args operatorAddress the params.`);
-            }
+    const target = addressToScVal(chain.contracts?.axelar_gas_service?.address);
+    const method = nativeToScVal('add_gas', { type: 'symbol' });
+    const params = nativeToScVal([
+        addressToScVal(sender),
+        nativeToScVal(messageId, { type: 'string' }),
+        addressToScVal(spender),
+        tokenToScVal(tokenAddress, tokenAmount),
+    ]);
 
-            const operator = Address.fromString(options.args).toScVal();
-            operation = contract.call('add_operator', operator);
-            break;
-        }
+    const operation = contract.call('execute', operator, target, method, params);
```

Comment:
Add gas is not called via the operators. Think about who calls it and how it should be used. See EVM script/contract for reference